README.yaml

```diff
@@ -0,0 +1,94 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-cloudfront-cdn
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-cloudfront-cdn
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-cloudfront-cdn.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-cloudfront-cdn"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-cloudfront-cdn.svg"
+    url: "https://github.com/cloudposse/terraform-aws-cloudfront-cdn/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-aws-cloudfront-s3-cdn"
+    description: "Terraform module to easily provision CloudFront CDN backed by an S3 origin"
+    url: "https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn"
+  - name: "terraform-aws-s3-log-storage"
+    description: "This module creates an S3 bucket suitable for receiving logs from other AWS services such as S3, CloudFront, and CloudTrail"
+    url: "https://github.com/cloudposse/terraform-aws-s3-log-storage"
+  - name: "terraform-aws-cloudtrail"
+    description: "Terraform module to provision an AWS CloudTrail and an encrypted S3 bucket with versioning to store CloudTrail logs"
+    url: "https://github.com/cloudposse/terraform-aws-cloudtrail"
+  - name: "terraform-aws-s3-website"
+    description: "Terraform module to provision S3-backed Websites"
+    url: "https://github.com/cloudposse/terraform-aws-s3-website"
+  - name: "terraform-root-modules-aws-docs"
```

Comment:
let's rewrite this to `terraform-root-modules/aws/docs`

Reply:
(it's just a friendly name. usually module names work, but in this case, we should refer to it some other way for clarity.)